src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -429,3 +434,8 @@
           text: >
             az acr build -r MyRegistry .
 """
+
+helps['acr image import'] = """
+    type: command
+    short-summary: Copies an image to the container registry from the specified container registry. Specify the source image either by login server or resource id.
+"""
```

Comment:
Consider improving the summary ...

Add examples for different cases:

 - Import from the same registry.
 - Import from a different registry in the same sub.
 - Import from a different registry in a different sub.
 - Import from Docker Hub (probably not supported currently).
 - ...